content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -12,10 +12,47 @@ If a payment is **not linked to an invoice or bill**, the customer has an outsta
 your company, or your company has an outstanding debit with a vendor. You can use those outstanding
 amounts to reduce unpaid invoices/bills.
 
+.. seealso::
+   - :doc:`Internal transfers <../../bank/misc/interbank>`
+   - :doc:`../../bank/reconciliation/use_cases`
+   - `Odoo tutorial on Bank Configuration <https://www.odoo.com/slides/slide/bank-configuration-1880?fullscreen=1>`_
+
+Invoices and bills statuses
+===========================
+
+Different types of status exist in Odoo to help you quickly identify the stage of your transactions.
+
+Invoices and bills creation statuses
+------------------------------------
+
+- :guilabel:`Draft`: Upon creation of a customer invoice or a vendor bill. The document is not
+  confirmed yet, and no journal entry gets created.
+- :guilabel:`Posted`: Once the invoice is confirmed, a journal entry is created.
+
+.. image:: recording/draft-posted.png
+   :align: center
+   :alt: See status information of a payment
+
+Invoices and bills status when registering payment
+--------------------------------------------------
+
+The customer invoice or vendor bill should be in the :guilabel:`Posted` status to register the
+payment. When clicking on :guilabel:`Register payment`, you can select the amount to pay and make a
```

Comment:
```suggestion
payment. When clicking on :guilabel:`Register payment`, select the amount to pay and make a
```

Reply:
Ignoring these changes so far as are part of another doc